README.md

```diff
@@ -235,15 +235,21 @@ Amber is a community effort and we want You to be part of it. [Join Amber Commun
 4. Push to the branch (git push origin my-new-feature)
 5. Create a new Pull Request
 
-## Amber Core Team
+## Amber Maintainers
 
-- [drujensen](https://github.com/drujensen) Dru Jensen - Maintainer
-- [eliasjpr](https://github.com/eliasjpr) Elias Perez - Maintainer
-- [elorest](https://github.com/elorest) Isaac Sloan - Maintainer
-- [fridgerator](https://github.com/fridgerator) Nick Franken - Maintainer
-- [marksiemers](https://github.com/marksiemers) Mark Siemers - Maintainer
-- [robacarp](https://github.com/robacarp) Robert Carpenter - Maintainer
+- Dru Jensen [drujensen](https://github.com/drujensen) 
+- Elias Perez [eliasjpr](https://github.com/eliasjpr) 
+- Isaac Sloan [elorest](https://github.com/elorest) 
+- Faustino Aguilar [faustinoaq](https://github.com/faustinoaq) 
+- Nick Franken [fridgerator](https://github.com/fridgerator)
+- Mark Siemers [marksiemers](https://github.com/marksiemers)
+- Robert Carpenter [robacarp](https://github.com/robacarp) 
 
+## Amber Contributors
+
+- Mike George [migeorge](https://github.com/migeorge)
+- Tatsujin Chin [c910335](https://github.com/c910335)
+- V. Elenhaupt [veelenga](https://github.com/veelenga)
```

Comment:
Thanks for adding me and guys on this list. But there are much more amber contributors which can be explored here: https://github.com/amberframework/amber/graphs/contributors

I think this section is useless in Readme.

Reply:
It might be better to link to that contributors page with a thank you in the README.

Eventually, this will get stale, and who is to say what level of contribution gets you on the README vs not.

Reply:
Done! 👍